### PR TITLE
Update migration page with v2 delegate requirements

### PR DESCRIPTION
**Pull request by torhector2**
Update migration page with v2 delegate requirements

TODOs:

- [x]  maybe delegator page needs to show migrated v2 delegates ???
- [x] ~Show v1 and v2 visual indications on delegates pages and list items ???~ done in separate pr

### CODE REVIEW

**Review by hernandoagf — APPROVED** [review]
Looking good so far!

File: modules/migration/components/MigrationBanner.tsx [hunk 1]
```diff
@@ -21,19 +21,25 @@ export function MigrationBanner(): React.ReactElement | null {
     isDelegatedToExpiredContract,
     isDelegateContractExpired,
     isDelegateContractExpiring,
-    isShadowDelegate
+    isShadowDelegate,
+    isDelegateContractV1,
+    isDelegatedToV1Contract
```
Comment (adamgoth) [inline modules/migration/components/MigrationBanner.tsx:26]: Minor thing but we make name consistent here, `V1Contract`. They are currently flipped around in each variable name.

**Review by adamgoth — APPROVED** [review]
Looking good so far. Left one small suggestion.

File: modules/migration/components/MigrationBanner.tsx (hunk 1 above)
Comment (torhector2) [reply modules/migration/components/MigrationBanner.tsx:26]: good catch

**Pull request merged**